.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 🐛 Bug
+about: Create a report to help us improve Blaze
+title: ''
+labels: ''
+assignees: ''
+---
+<!--
+Before reporting a bug, please check for existing or closed issues
+first and read the instructions for filing a bug report:
+https://github.com/meteor/meteor/blob/devel/CONTRIBUTING.md#reporting-a-bug-in-meteor
+
+### This bug report should include:
+- [ ] A short, but descriptive title. The title doesn't need "Blaze" in it.
+- [ ] The version of Blaze showing the problem.
+- [ ] The last version of Blaze where the problem did _not_ occur, if applicable.
+- [ ] The operating system you're running Blaze on.
+- [ ] The expected behavior.
+- [ ] The actual behavior.
+- [ ] A **simple** reproduction! (Must include the Github repository and steps to reproduce the issue on it.)
```

Comment:
We should change `(Must include the Github repository and steps to reproduce the issue on it.)` to

> (Must either include a minimal code example or a link to a  Github repository with steps to reproduce the issue on it.)